src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -156,7 +156,7 @@ <h2>{{ addon.name }}</h2>
     <h3>{{ distro_tag(amo.RELEASE_CHANNEL_LISTED)}} {{ _('Listing visibility') }}</h3>
     <div class="item" id="addon-current-state">
       <div class="item_wrapper">
-        {% set owner = check_addon_ownership(request, addon, dev=True) %}
```

Comment:
fun fact! Despite the intention to only allow owners of add-ons to change the checkbox this never worked correctly and allowed developer-role users to change the checkbox (though the view would have enforced the check and thrown an error when the change was saved)

Reply:
Although now I'm wondering if the test was just wrongly worded because the view doesn't enforce the check.
@jvillalobos should non-owners (developer role) be able to change the add-on user_disabled status?

Reply:
Yes.